Source/WTF/wtf/glib/FileSystemGlib.cpp

```diff
@@ -66,7 +66,7 @@ CString currentExecutablePath()
     ssize_t result = readlink("/proc/self/exe", readLinkBuffer, PATH_MAX);
     if (result == -1)
         return { };
-    return CString(readLinkBuffer, result);
+    return CString({ readLinkBuffer, static_cast<size_t>(result) });
```

Comment:
Can we omit `CString()` here?

Reply:
Maybe not, I don’t see how it figures out to use `std::span`.

Reply:
`readLinkBuffer` is not `const` so it's the same issue as above.